docs/chapter1/1introduction.md

```
@@ -16,52 +16,67 @@ slug : /
+  title="CircuitVerse Simulator - Interactive Demo"
+/>
+
+Inspired by Logisim, CircuitVerse is an open-source educational tool for designing and visualizing digital logic circuit simulations in the cloud.
+
+Users—including educators, students, electronics engineers, and hobbyists—can experiment with digital circuit simulations directly from their browsers to explore various "what-if" scenarios. Users can share their creations using unique circuit URLs or fork contributions from the CircuitVerse community to satisfy their curiosity.
+
+Figure 1.1 shows a screenshot of a ripple carry adder simulation built using CircuitVerse.
+
+![Ripple Carry Adder simulation showing input bits, full adder components, sum outputs, and carry chain in the CircuitVerse simulator](/img/img_chapter1/1.1.png)
+
+*Figure 1.1: Ripple Carry Adder simulation built using CircuitVerse*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)